problems/removeVowels.js

```diff
@@ -6,7 +6,8 @@
 * @returns {string} - Returns a new string without any vowels.
 */
 
-function removeVowels() {
+function removeVowels(str) {
+    return str.replace(/[aeiouAEIOU]/g,"")
```

Comment:
This answer is very very slick, but make sure you understand 100% how this RegEx is working and could replicate this answer on an interview.